src/lib/Libnet/hnls.c

```diff
@@ -574,53 +574,162 @@ free_if_info(struct log_net_info *ni)
 
 /**
 * @brief
-	Get a list of all IPs
+	Get a list of all IPs (ipv4) for a given hostname
+*
+* @return
+*	Comma separated list of IPs in string format
 *
-* @return 
-*	Comma separated list of ips in string format
-* 
 * @par Side Effects:
 *	None
 *
 * @par MT-safe: Yes
 *
-* @param[out]   msg - error message returned if system calls not successful
+* @param[in]	host        - hostname of the current host to resolve IPs for
+* @param[out]   msg_buf     - error message returned if system calls not successful
+* @param[in]    msg_buf_len - length of the message buffer passed
+*
+*/
+static char *
+get_host_ips(char *host, char *msg_buf, size_t msg_buf_len)
+{
+	struct addrinfo *aip, *pai;
+	struct addrinfo hints;
+	int rc = 0;
+	char buf[NETADDR_BUF] = {'\0'};
+	int count = 0;
+	char *nodenames = NULL;
+	char *tmp, *p;
+	int len, hlen;
+
```

Comment:
The indenting of variables is somewhat vague in our standard.  It says we should indent them "normally".  To me a "normal" indent is the indent in front of the code.  This would mean you'd indent before the type, not after.  We probably should  clarify it.  Whether or not we indent after the type should be clear in the standard.  All of the variable declarations should look the same across the code.

Reply:
I will try to align. I do not think we should check msg_buf for NULL, since it is expected usage to pass a non-null variable. The accompanying parameter parameter is specifying the size of the buffer anyway. Besides what do i do if i find that it is NULL, i can't even return an error message! So, the caller has to understand the API and call properly.

Reply:
About indenting, I agree with @bhroam . Besides, I have followed the style in that c file itself, so it would not be useful to try to realign everything. (In general, I do not see aligning after the type to be any more beautiful or readable either)